vyxal/__main__.py

```diff
@@ -0,0 +1,594 @@
+import os
+import secrets
+import sys
+
```

Comment:
why is this file here?

Reply:
So that it can be run with `python -m vyxal`. I guess calling it `vyxal.py` would have worked too, but I wasn't sure if that meant that any `import vyxal.asdf` import would require loading `vyxal.py` first.

Reply:
Why did you copy the contents of `interpreter.py` though? Just import it

Reply:
Will the `if __name__ == "__main__":` stuff still be triggered, though?

Reply:
good point - how about you symlink `__main__.py` to `interpreter.py` - that'll work without duplicating the code.

Reply:
Er, I'm not sure how to do that. How about a plain `main` method inside `interpreter.py` instead?

Reply:
on Linux or macOS: `ln -s interpreter.py __main__.py`; on Windows: [instructions](https://t.pxeger.com/xs.html?t=we+do+a+little+trolling&v=get+a+better+OS+lmao)

Reply:
bruh